styleguide/_themes/derek/scss/modifiers.scss

```diff
@@ -45,6 +45,13 @@ $body-bg:  #fff !default;
 }
 
 //centered-divs
+@mixin center-block {
+  display: block;
+  float: none;
+  margin: 0 auto;
+}
+
+//making sure to support the class
 .center-block {
```

Comment:
If you're gonna do this, should `.center-block` then include this mixin instead?